playbooks/google.yml

```diff
@@ -7,26 +7,29 @@
 
   vars:
     zones:
-      "1": "us-central1-a"
-      "2": "us-central1-b"
-      "3": "us-central1-c"
-      "4": "us-central1-f"
-      "5": "us-east1-b"
-      "6": "us-east1-c"
-      "7": "us-east1-d"
-      "8": "us-west1-a"
-      "9": "us-west1-b"
-      "10": "europe-west1-b"
-      "11": "europe-west1-c"
-      "12": "europe-west1-d"
-      "13": "asia-east1-a"
-      "14": "asia-east1-b"
-      "15": "asia-east1-c"
-      "16": "asia-northeast1-a"
-      "17": "asia-northeast1-b"
-      "18": "asia-northeast1-c"
-      "19": "asia-southeast1-a"
-      "20": "asia-southeast1-b"
+      "1":  "us-central1-a"
+      "2":  "us-central1-b"
+      "3":  "us-central1-c"
+      "4":  "us-central1-f"
+      "5":  "us-east1-b"
```

Comment:
these all seem to be mapped backwards. "us-east1-b" = (Northern Virginia A) in the vars_prompt.